pkg/crd/crdloader/crdloader.go

```diff
@@ -155,7 +156,27 @@ func formatGVK(group, version, kind string) string {
 }
 
 func LoadCRDs() ([]apiextensions.CustomResourceDefinition, error) {
-	crdsRoot := repo.GetCRDsPath()
+	return loadCRDs(repo.GetCRDsPath())
+}
+
+func LoadAllCRDs() ([]apiextensions.CustomResourceDefinition, error) {
+	results := make([]apiextensions.CustomResourceDefinition, 0)
+	loadFuncs := []func() ([]apiextensions.CustomResourceDefinition, error){LoadCRDs, loadOperatorCRDs}
+	for _, loadFunc := range loadFuncs {
+		crds, err := loadFunc()
+		if err != nil {
+			return nil, err
+		}
+		results = append(results, crds...)
+	}
+	return results, nil
+}
+
```

Comment:
nit: It would be beneficial to have comments explaining the differences between LoadCRDs and LoadAllCRDs, as their names are similar and may lead to confusion.

IIUC LoadAllCRDs includes CRDs + OperatorCRDs

Reply:
Done.